lib/TopStoriesFeed.jsm

```diff
@@ -113,13 +113,18 @@ this.TopStoriesFeed = class TopStoriesFeed {
     this.store.dispatch(shouldBroadcast ? ac.BroadcastToContent(action) : ac.AlsoToPreloaded(action));
   }
 
-  doContentUpdate(shouldBroadcast) {
+  doContentUpdate({stories, topics}, shouldBroadcast) {
```

Comment:
Probably good to document the method to be clear why it exists vs the previous behavior -- namely a single place to update both stories and topics instead of trying to update them individually.